### PR TITLE
[SM]: StartPage implementation

### PR DESCRIPTION
**Pull request by sureshhmrc**

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -171,4 +172,11 @@ claimPeriod.Oct2021 = October 2021
 claimPeriod.Nov2021 = November 2021
 claimPeriod.Dec2021 = December 2021
 claimPeriod.error.required = Select one value
-claimPeriod.error.invalid = Enter a real claim period start
\ No newline at end of file
+claimPeriod.error.invalid = Enter a real claim period start
```
Comment (mikail-khan) [inline conf/messages.cy:175]: Change this to  "Choose a claim period"

File: conf/messages.cy [hunk 2]
```diff
@@ -171,4 +172,11 @@ claimPeriod.Oct2021 = October 2021
...
+
+
+startPage.title = Coronavirus Job Support Scheme calculator
```
Comment (mikail-khan) [inline conf/messages.cy:178]: Capitalise first letter of calculator - Ramon has it capitalised in this wireframes

File: conf/messages.en [hunk 3]
```diff
@@ -171,4 +172,10 @@ claimPeriod.Oct2021 = October 2021
 claimPeriod.Nov2021 = November 2021
 claimPeriod.Dec2021 = December 2021
```
Comment (mikail-khan) [inline conf/messages.en:173]: why are these here?

Comment (sureshhmrc) [reply conf/messages.en:173]: these are messages displayed as radio buttons

**Comment by mikail-khan** [comment]
could you also set the back link in the ClaimPeriod page to refer back to this start page?

File: conf/messages.cy (hunk 2 above)
Comment (sureshhmrc) [reply conf/messages.cy]: fixed

File: conf/messages.cy (hunk 1 above)
Comment (sureshhmrc) [reply conf/messages.cy]: fixed

File: conf/messages.en (hunk 3 above)
Comment (mikail-khan) [reply conf/messages.en:173]: we only need Nov2021 for now in that case
